Tools/autotest/models/Callisto.json

```diff
@@ -57,5 +57,9 @@
     "mdrag_coef" : 0.10,
 
     # number of motors
```

Comment:
please please add support for '#' for comments, I know it is non-standard, but it is extremely useful

Reply:
> please please add support for '#' for comments, I know it is non-standard, but it is extremely useful

I swiped the existing "remove #" code - it is present, and does parse Callisto.json which has that sort of comment in it.

IOW - the support should already be there unless I'm missing something

Reply:
ahh, sorry, saw the red in github markup and thought it was removing the comment line!